Sprint-3/revise/implement/repeat.test.js

```diff
@@ -23,3 +23,41 @@
 // Given a target string str and a negative integer count,
 // When the repeat function is called with these inputs,
 // Then it should throw an error or return an appropriate error message, as negative counts are not valid.
```

Comment:
I do know of an alternate solution to this challenge, but I'm not sure if your syllabus covers it.